packages/core/core-flows/src/payment/workflows/capture-payment.ts

```diff
@@ -13,6 +15,12 @@ export const capturePaymentWorkflow = createWorkflow(
     }>
   ): WorkflowData<PaymentDTO> => {
     const payment = capturePaymentStep(input)
+
```

Comment:
Not related to this PR, but as you added the captured event, do you mind to add the refunded as well?

Reply:
sure thing 🤞🏻

Reply:
Added @carlos-r-l-rodrigues